scss/partials/_roster.scss

```diff
@@ -1,3 +1,52 @@
+.jsxc-collapsible {
+  background-color: #777;
```

Comment:
With the latest release I introduced css variables to allow to easily adapt the color scheme. Therefore those vars should be used here as well.

Reply:
using var .. now